### PR TITLE
DEVENGAGE-1121 Updated versions

### PR DESCRIPTION
**Pull request by Zino-ctrlZ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,12 +24,12 @@ import { DxTextbox } from 'genesys-react-components';
 import { GenesysDevIcons } from 'genesys-dev-icons';
 
 export default function App() {
-	const inputRef = useRef<HTMLInputElement>(null);
+	const inputRef = useRef < HTMLInputElement > null;
 	return (
-		<div className='app'>
+		<div className="app">
```
Comment (timsmithgenesys) [inline README.md:29]: Make sure you have the Prettier extension installed in vs code and format this file. The `.prettierrc.yml` file in the root of the repo defines the styles that should be used.

Comment (Zino-ctrlZ) [reply README.md:29]: I've got single quotes on prettier checked, so I'm not sure why it still sets some values to double quotes

Comment (timsmithgenesys) [reply README.md:29]: You might not have prettier set as the default formatter in vs code https://stackoverflow.com/a/63974154

Comment (timsmithgenesys) [reply README.md:29]: @Zino-ctrlZ I don't know what's going on with prettier. It's behaving differently for me as well. Today the beast wins... we shall live to fight this battle another day. :)

**Review by timsmithgenesys — APPROVED** [review]

**Pull request merged**